optimize/components/userguide/user-permissions.md

```diff
@@ -4,20 +4,16 @@ title: User permissions
 description: "To share a collection of reports and dashboards with other users, add them to the collection."
 ---
 
-By default, if you create a collection, only you can access the collection and the contents within. To share a collection of reports and dashboards with other users, add them to the collection.
+By default, if you create a collection, only you can access the collection and the contents within. To share a collection with other users, add them to the collection.
 
 ![users and user groups](./img/users.png)
 
 You are automatically assigned the manager role when creating a new collection. There can be multiple managers for a collection. However, there must be at least one manager for every collection. Managers can do the following:
 
-- Add, edit, and remove dashboards and reports to the collection.
-- Edit the collection name and delete the collection using the context menu in the header.
-- Add, edit, and remove other users and user groups to collections via the collection's **Users** tab.
+User permissions in collections involve managers, editors, and viewers. Managers control collection content, including adding/removing dashboards and reports. Editors can modify content but not collection details, while viewers can only observe and copy. Add users/groups to collections via the **Users** tab, assigning roles that determine access rights.
```

Comment:
I would say that the key distinction is more that Managers can manage which users and data sources can be used within a collection, while editors can only create content based on those data sources.

@toco-cam might have input here

Reply:
Fully agree with: "Managers can manage which users and data sources".